docs/components/connectors/out-of-the-box-connectors/google-maps-platform.md

```diff
@@ -0,0 +1,106 @@
+
+:::note
+We advise you to keep your authentications and secrets data safe and avoid exposing it in the BPMN XML file by creating a secret:
+
+1. Follow our [guide for creating secrets](../../console/manage-clusters/manage-secrets.md).
+2. Name your secret (i.e `GOOGLE_MAPS_PLATFORM_API_KEY`) so you can reference it later in the Connector.
+   :::
+
+## Operation types
+
+### Validate address
+
+This operation allows you to validate an address and its components, standardize the address for mailing, and determine the best known geocode for it.
+To use this operation, enable the [Google Address Validation API](https://developers.google.com/maps/documentation/address-validation/overview). See the [official documentation](https://developers.google.com/maps/documentation/address-validation/cloud-setup) for more information on enabling the Google API.
+
+### Get place ID
+
+This operation allows you to get the Google Maps place ID by address.
+To use this operation, enable the [Google Places API](https://developers.google.com/maps/documentation/places/web-service). See the [official documentation](https://developers.google.com/maps/documentation/places/web-service/get-api-key) for more information on enabling the Google API.
+
+### Calculate distance
+
+This operation allows you to calculate a distance between two place IDs.
+To use this operation, enable the [Google Directions API](https://developers.google.com/maps/documentation/directions). See the [official documentation](https://developers.google.com/maps/documentation/directions/get-api-key) for more information on enabling the Google API.
+
+## Usage
+
+### Address validation, formatting, getting postal address
+
+1. Select **Validate Address** from the **Operation type** dropdown in the **Operation** section.
+2. Populate the **Authentication** section as described in the [respective section](#authentication).
+3. (Optional) In the **Input** section, set **Region Code** (i.e `US`). You can find supported region codes [here](https://developers.google.com/maps/documentation/address-validation/coverage).
+4. (Optional) In the **Input** section, set **Locality**, an incorporated city or town political entity (i.e `Mountain View`).
+5. In the **Input** section, set **Address**, an incorporated city or town political entity (i.e `1600 Amphitheatre Pkwy`).
+6. In the **Output** section set **Result Variable** or **Result Expression**. See the [response mapping documentation](/docs/components/connectors/use-connectors.md#response-mapping) to learn more.
+7. Get from response needed information. Find a full example of the response [here](https://developers.google.com/maps/documentation/address-validation/requests-validate-address#address_validation_response). For example, if you want to get postal address and formatted address, you can set to **Result Expression** next FEEl Expression :
```

Comment:
Can you explain what you are trying to say here? Particularly "Get from response needed information."